assets/src/application/ui/layout/entityList/RegistrationsLink/index.tsx

```diff
@@ -14,7 +14,12 @@ interface Props {
 const RegistrationsLink: React.FC<Props> = ({ href, title }) => {
 	return (
 		<Tooltip title={title}>
-			<a href={href} className={'ee-editor-details-reg-url-link'} target={'_blank'} rel={'noopener norefferer'}>
+			<a
```

Comment:
@tn3rb I think we might better have a `Clickable` component here which can be a wrapper that applies button styles to an anchor.
An example: https://reakit.io/docs/clickable/